contracts/BasicNFT-v2.cdc

```diff
@@ -0,0 +1,81 @@
+/* 
+*
+*  This is an basic implementation of a Flow Non-Fungible Token using the V2 standard.
+*  It shows that a basic NFT can be defined in very few lines of code (less than 100 here)
+*
+*  Unlike the `ExampleNFT-v2` contract, this NFT illustrates a minimal implementation
+*  of an NFT that is now possible with the NFT standard since Events, collections,
+*  and other old requirements are not required any more.
+* 
+*  It also includes minimal metadata to showcase the simplicity
+*   
+*/
+
+import NonFungibleToken from "./NonFungibleToken-v2.cdc"
+import MetadataViews from "./MetadataViews.cdc"
+
+pub contract BasicNFT {
+
+    /// The only thing that an NFT really needs to have is this resource definition
+    pub resource NFT: NonFungibleToken.NFT, MetadataViews.Resolver {
+        /// Arbitrary trait mapping metadata
+        access(self) let metadata: {String: AnyStruct}
+    
+        init(
+            metadata: {String: AnyStruct},
+        ) {
+            self.metadata = metadata
+        }
+
+        /// Gets the ID of the NFT, which here is the UUID
+        pub fun getID(): UInt64 { return self.uuid }
```

Comment:
Can this be defaultly implemented in the NFT standard?

Reply:
yes, good catch